From d272ce8ec22f5bdb9adccd0eb46b12e88e958a7e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:48:53 +0800
Subject: [PATCH] Revert "drivers: usb: gadget: Add helper function for
 installing ACM gadget function."

This reverts commit 5502501ffd887bfe047c1f3e1e36ac06b0f0e756.
---
 drivers/usb/gadget/f_acm.c |  9 ---------
 drivers/usb/gadget/f_acm.h | 24 ------------------------
 2 files changed, 33 deletions(-)
 delete mode 100644 drivers/usb/gadget/f_acm.h

diff --git a/drivers/usb/gadget/f_acm.c b/drivers/usb/gadget/f_acm.c
index 7dd1a8bbe382..7953948bfe4a 100644
--- a/drivers/usb/gadget/f_acm.c
+++ b/drivers/usb/gadget/f_acm.c
@@ -761,12 +761,3 @@ int __init acm_bind_config(struct usb_configuration *c, u8 port_num)
 		kfree(acm);
 	return status;
 }
-
-int __init acm_function_add(struct usb_composite_dev *cdev,
-	struct usb_configuration *c)
-{
-	int ret = acm_bind_config(c, 0);
-	if (ret == 0)
-		gserial_setup(c->cdev->gadget, 1);
-	return ret;
-}
diff --git a/drivers/usb/gadget/f_acm.h b/drivers/usb/gadget/f_acm.h
deleted file mode 100644
index 8e27b3440635..000000000000
--- a/drivers/usb/gadget/f_acm.h
+++ /dev/null
@@ -1,24 +0,0 @@
-/*
- * Gadget Driver for Android ACM
- *
- * Copyright (C) 2009 Motorola, Inc.
- * Author:
- *
- * This software is licensed under the terms of the GNU General Public
- * License version 2, as published by the Free Software Foundation, and
- * may be copied, distributed, and modified under those terms.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- */
-
-#ifndef __F_ACM_H
-#define __F_ACM_H
-
-int acm_function_add(struct usb_composite_dev *cdev,
-	struct usb_configuration *c);
-
-#endif /* __F_ACM_H */
-- 
2.35.3

